app/main.py

```diff
@@ -1 +1,42 @@
-# write your code here
+from __future__ import annotations
+
+
+class Animal:
+    alive = []
+
+    def __init__(self,
+                 name: str,
+                 health: int = 100,
+                 hidden: bool = False
+                 ) -> None:
```

Comment:
read please this message and fix everywhere ;)
https://mate.academy/chats/244801?selectedMessageId=3129519